src/Transport/Creation/AzureServiceBusSubscriptionCreator.cs

```diff
@@ -31,15 +31,22 @@ public AzureServiceBusSubscriptionCreator(ReadOnlySettings settings)
                     MaxDeliveryCount = setting.GetOrDefault<int>(WellKnownConfigurationKeys.Topology.Resources.Subscriptions.MaxDeliveryCount),
                     RequiresSession = setting.GetOrDefault<bool>(WellKnownConfigurationKeys.Topology.Resources.Subscriptions.RequiresSession),
                     
+                    // TODO: default forwarding for subscriptions is not clear... need to review this
                     ForwardTo = setting.GetConditional<string>(subscriptionName, WellKnownConfigurationKeys.Topology.Resources.Subscriptions.ForwardTo),
```

Comment:
I think we need to remove this configuration option, if forwarding is set the StandardTopo would not be able to read from the sub, if set on ForwardTopo it would not behave the way we expect it to, so I suggest to get rid of this setting for subscriptions